e2e/cypress/support/api_commands.js

```diff
@@ -662,6 +662,20 @@ Cypress.Commands.add('promoteUser', (userId) => {
     cy.externalRequest({user: admin, method: 'post', baseUrl, path: `users/${userId}/promote`});
 });
 
+/**
+ * Add a user to a team directly via API
+ * @param {String} teamId - The team ID
+ * @param {String} userId - The user ID
+ * All parameter required
+ */
+
+Cypress.Commands.add('addUserToTeam', (teamId, userId) => {
```

Comment:
Might want to update from master since this api command was added in this PR https://github.com/mattermost/mattermost-webapp/pull/5831

Reply:
@josephbaylon this one is little different. An external request is made as sysadmin when the user is already on a page.

Reply:
@srkgupta I see. I think it would be a little confusing later. I did a check on the codebase and all commands/task that use `cy.externalRequest(...)` and/or `cy.task('externalRequest', ...)` require admin login. Not necessary for this PR, but might be good to start prefixing externalRequests either `adminXXX` or `externalXXX` just so it is differentiated from `apiXXX`.

Reply:
+1 to Joseph. I'm leaning towards `external` prefix. Also, please don't add any command to `api_commands.js`, we'll deprecate it soon. We're migrating/organizing commands into `support/api/*.js`. For this command, it might not fit to that folder and probably better to create a new one, like `support/external_commands.js`. Lastly, please add typings `support/external_commands.d.ts`

Reply:
Thanks @josephbaylon and @saturninoabril. There are some other APIs following the same naming conventions which might also need to be updated and is beyond the scope of this PR. Can we do the cleanup as a separate PR instead.

Reply:
I agree on other APIs and indeed it should be on a separate PR. But since you're introducing a new custom command, I suggest to start the recommended way here.

Reply:
@saturninoabril and @josephbaylon, Updated the code as suggested. Please recheck.